index.html

```diff
@@ -82,6 +83,20 @@
         var(--highlight-color);
     }
 
+    [dir=rtl] gaia-list.contacts .face {
```

Comment:
TBH I don't know if we can get any benefit from fixing RTL issue in example only. if these item is commonly used in list, why not moving these styling in gaia-list?
```
::content .photo-at-right {
  left: auto;
  right: 16px;
}

::content .photo-at-right {
  left: auto;
  right: 16px;
}
[dir=rtl] ::content .photo-at-right {
  left: 16px;
  right: auto;
}
```